exec/java-exec/src/test/java/org/apache/drill/BaseTestQuery.java

```diff
@@ -379,6 +379,16 @@ protected static void parseErrorHelper(final String testSqlQuery) throws Excepti
     errorMsgTestHelper(testSqlQuery, UserBitShared.DrillPBError.ErrorType.PARSE.name());
   }
 
+  /**
+   * Utility method which tests given query produces a {@link UserException}
+   * with {@link org.apache.drill.exec.proto.UserBitShared.DrillPBError.ErrorType}
+   * being DrillPBError.ErrorType.VALIDATION the given message.
+   * @param testSqlQuery Test query
+   */
+  protected static void validationErrorHelper(final String testSqlQuery) throws Exception {
```

Comment:
Can you use ```thrownException.expect(new UserExceptionMatcher(VALIDATION, "..."));``` in unit tests that use this method (rather than defining and using this method)?